content/influxdb/cloud-serverless/reference/sql/functions/time-and-date.md

```diff
@@ -113,7 +179,182 @@ ORDER BY time DESC
 {{% /expand %}}
 {{< /expand-wrapper >}}
 
-### date_trunc
+## date_bin_gapfill
+
+Calculates time intervals and returns the start of the interval nearest to the specified timestamp.
+If no rows exist in a time interval, a new row is inserted with a `time` value
+set to the interval start time, all queried
+[primary key](/influxdb/cloud-serverless/reference/glossary/#primary-key)
+columns populated, and null values in aggregate columns.
```

Comment:
I don't think it's quite right to say "primary key" here (which I take to mean the tag columns and the time column). It would be more accurate to say that all the columns from the `GROUP BY` clause will be populated.

While it will often be the case that these are the primary key columns for the table, it's not necessarily so.

Reply:
Perfect, thanks @wolffcm. Made this update.